plugins/catalog-backend-module-github/src/processors/GithubMultiOrgReaderProcessor.ts

```diff
@@ -130,12 +133,19 @@ export class GithubMultiOrgReaderProcessor implements CatalogProcessor {
           client,
           orgConfig.name,
           tokenType,
-          orgConfig.userNamespace,
         );
-        const { groups, groupMemberUsers } = await getOrganizationTeams(
+        const { groups } = await getOrganizationTeams(
           client,
           orgConfig.name,
-          orgConfig.groupNamespace,
+          async (team, ctx): Promise<GroupEntity | undefined> => {
+            const result = await defaultOrganizationTeamTransformer(team, ctx);
```

Comment:
not the plugged-in one if there is one?

Reply:
I have added the config here to enable transformers for this processor.